src/components/welcome/MyAccounts/useTrackedSafesCount.ts

```diff
@@ -0,0 +1,21 @@
+import { AppRoutes } from '@/config/routes'
+import { OVERVIEW_EVENTS, trackEvent } from '@/services/analytics'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+import type { SafeItems } from './useAllSafes'
+
+const useTrackSafesCount = (ownedSafes: SafeItems | undefined, watchlistSafes: SafeItems | undefined) => {
+  const [isTracked, setIsTracked] = useState(false)
```

Comment:
This is not global tho, is it? It will run as many times as this hook runs. I.e. on every Safe List render.

Reply:
you mean like this?

```
let isTracked = false

const useTrackSafesCount = (ownedSafes: SafeItems | undefined, watchlistSafes: SafeItems | undefined) => {
  const router = useRouter()
  const isLoginPage = router.pathname === AppRoutes.welcome.accounts

  useEffect(() => {
    if (watchlistSafes && ownedSafes && isLoginPage && !isTracked) {
     ...
      isTracked = true
    }
  }, [isLoginPage, ownedSafes, watchlistSafes])
}
```

Reply:
Yes, please test it, though. In some build systems imported modules are functions that are executed every time on import.

Reply:
Tested, and the only time it sends them again is on browser reload. Navigating back to the login page doesn't send them again, which I think is what we want here.

Reply:
Did you test it locally or on the PR preview? Dev & prod build are quite different.

Reply:
With the PR preview link